reference/audit_logs.rst

```diff
@@ -0,0 +1,241 @@
+JAAS: Understanding Audit Logs
+==============================
+
+Introduction
+------------
+
+JIMM provides audit logging functionality, tracking all requests/responses into the system.
+This gives administrators of JIMM the ability to audit changes at a very granular level.
+
+All requests to controllers and models are logged and can enable an analysis into why the state
+of the underyling Juju estate has changed.
+
+Audit logs are currently enabled by default without any config to disable their collection.
+The logging retention period is configurable and allows for audit logs to be stored for a
+variable amount of time depending on your requirements and storage constraints.
+
+Filtering audit logs is also possible along a variety of fields including but not limited to the
+user executing the request, model name and command.
+
+Finally, access to audit logs is, by default, only afforded to JIMM superuers i.e. administrators of JIMM.
+Read access to audit logs can be granted to other users via jimmctl, see :doc:`/tutorial/group_management`
+
+Filter Logs
+-----------
+Querying for audit logs is most readily done via the jimmctl CLI tool.
+
+Basics
+~~~~~~
+
+``jimmctl audit-events --help``
+
+will show you a list of all available filter options
```

Comment:
I would do something like:
```
To list all available filtering options:
<command>
```

I think it's clearer

Reply:
Agreed